build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
     implementation('org.apache.ivy:ivy:2.4.0')
     api('org.assertj:assertj-core:3.4.1')
     implementation('org.apache.commons:commons-lang3:3.5')
+    implementation('org.springframework:spring-core:5.2.9.RELEASE')
```

Comment:
Looks like `api` should  be used instead of `implementation` if this library is needed in run-time. See https://github.com/jenkinsci/JenkinsPipelineUnit/issues/209